src/plugin.rs

```diff
@@ -133,6 +129,18 @@ impl<A: Actionlike> Plugin for InputManagerPlugin<A> {
                         .after(UiSystem::Focus)
                         .after(InputSystem),
                 );
+
+                app.add_systems(
```

Comment:
I think this will be cleaner if we move it above the feature-gated code.

Reply:
I moved it above the `configue_sets()` to pair it with the other `add_systems()`